features/2025-07-10-offline-install-feature-spec.md

```diff
@@ -0,0 +1,386 @@
+# Offline Installation Feature Spec
+
+[@zachcasper](https://github.com/zachcasper)
+
+## Summary
+
+Many large organizations have mature security practices where all software entering the organization's environment are controlled. Typically all software packages are scanned for security vulnerabilities prior to being stored internally. Access to non-authorized software packages are restricted. 
+
+Radius must support these organizations' requirements by making it easy to install, configure, and upgrade Radius while offline, or in *air gapped* environments.
+
+### Goals
+
+* Enable platform engineers to install and upgrade Radius while not connected to the internet
+* Enable platform engineers to control what versions of all require software components are installed
+* Specifically, improve the way Terraform is installed, configured, and upgraded
+* Increase the transparency of what software is required to run Radius
+
+### Non-Goals (out of scope)
+
+* Enhancing the installation experience in other ways
+
+## Inventory of software components
+
+Radius makes the assumption that the installation environment has full access to the internet and installs many software components on behalf of the user. Below is an inventory:
+
+| #    | Component                                                    | As-Is Distribution                                           | To-Be Online Distribution                                    | To-Be Offline Distribution                             |
+| ---- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------ |
+| 1a   | `rad` CLI binary                                             | [install.sh](https://raw.githubusercontent.com/radius-project/radius/main/deploy/install.sh) script | No change                                                    | Manually installed                                     |
+| 1b   | `rad-bicep` binary                                           | [install.sh](https://raw.githubusercontent.com/radius-project/radius/main/deploy/install.sh) script | No change                                                    | Manually installed                                     |
```

Comment:
I think this is just the bicep binary, i.e. https://github.com/Azure/bicep/releases/